src/cli/mesh.js

```diff
@@ -0,0 +1,94 @@
+import settings from '../../settings';
+
+function meshCommand() {
+	if (!meshCommand._instance) {
+		const MeshCommand = require('../cmd/mesh').MeshCommand;
+		meshCommand._instance = new MeshCommand(settings);
+	}
+	return meshCommand._instance;
+}
+
+export default ({ commandProcessor, root }) => {
+	const mesh = commandProcessor.createCategory(root, 'mesh', 'Manage mesh networks');
+
+	commandProcessor.createCommand(mesh, 'create', 'Create a new network', {
+		// TODO: Provide descriptions for positional arguments?
+		params: '<network name> <device>',
```

Comment:
Let's use snake_case for the parameters so they can be referenced using `params.network_name` in later code.